gms/api/src/main/pegasus/com/linkedin/dataprocess/DataProcess.pdl

```diff
@@ -0,0 +1,34 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.DataProcessUrn
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.Uri
+import com.linkedin.common.VersionTag
+
+/**
+ * Spec. for a process. A collection of data conforming to a single schema that can evolve over time.
+ */
+record DataProcess includes DataProcessKey, ChangeAuditStamps, VersionTag {
```

Comment:
No need to include `ChangeAuditStamps` & `VersionTag`

Reply:
Removed ChangeAuditStamps & VersionTag